### ISSUE TITLE
Adjust oco order trading panel ui

### ISSUE THREAD

**Issue by lowwor** [gruelbox/orko#403]
Currently, there are two row in the oco trading panel, one for above and one for below. But mosly we just use one of them because the order direction is the same. So we can change this to one row and add two button ( >,< or above,below) on the left of the price, it's easier to understand and easier to use. And it's easier to implement amount in btc value for it.

**Comment by badgerwithagun** [comment]
Actually I think we're just sort of abusing the OCO panel.  It's intended to place two orders (a take profit and a stop-limit), only one of which will fill.

You can only specify one order if you wish, but that's mostly because it was a quick way for me to implement "soft" limit orders without any special code!

What I think we should do is add an "on exchange" checkbox in the Limit Order panel (as on the stop panel).  If that is set, we create a one-sided OCO (again, as we do on the stop panel).

Then add validation in the OCO panel forcing you to specify both sides, just to avoid confusion.

**Comment by lowwor** [comment]
Yeah, but there are some difference here. The ladder buy should use the low oco, stoploss should use the low oco and the take profit use the high oco. Need something to set it up.

**Comment by badgerwithagun** [comment]
I see.  I don't want to lose what OCO already does though - I use this a lot (set both a take-profit and a stop-loss, and one cancels the other). 

Why does my proposal not work for you?  The effect would be the same: just use the **Limit** or **Stop** Panels with "on exchange" turned off.  If I understand you correctly, this is exactly what you want.  It would create exactly the same orders (one-sided OCO jobs) with exactly the same UI behaviour (if you add the BTC amount to the limit and stop panels, you get it for free).

**Comment by lowwor** [comment]
Yeah, we can do it in stop panel. But we need to add a option there to achieve what I want, need a place to choose if it's a oco high job or oco low job. Because stop limit order panel just use high job for buy and low job for sell.
